code/script.js

```diff
@@ -30,18 +33,141 @@ const showMessage = (message, sender) => {
-// This means the greeting function will be called one second after the website is loaded.
+// Bot answer based up on selection. I added template literal: `${coffeeOption} so the bot begin answer with the option that user choosed.
+// Added 3 buttons of size alternatives (105,106,107). Same code as for hot and cold button in the start.
+const handleCoffeeOptionInput = (coffeeOption) => {
+  option = coffeeOption;
+  console.log(coffeeOption);
+  showMessage(`${coffeeOption}, excellent choice! What size do you want?`, 'bot');
+  inputWrapper.innerHTML = `
+    <button id="smallBtn">Small</button>
+    <button id="mediumBtn">Medium</button>
+    <button id="largeBtn">Large</button>
+  `
+  //When user click on size button it triggers answer from user to bot: small, medium or large
+  document.getElementById('smallBtn').addEventListener('click', () => {
+    showMessage('Small', 'user'); //User answer "small"
+    setTimeout(() => handleCoffeeSizeInput('Small'), 1000);
+  });
+
+  document.getElementById('mediumBtn').addEventListener('click', () => {
+    showMessage('Medium', 'user'); //User answer "medium"
+    setTimeout(() => handleCoffeeSizeInput('Medium'), 1000);
+  });
+
+  document.getElementById('largeBtn').addEventListener('click', () => {
+    showMessage('Large', 'user'); //User answer "large"
+    setTimeout(() => handleCoffeeSizeInput('Large'), 1000);
+  });
+}
+
+//Bot answer based up on selection. I added two template literals: ${coffeeSie} and ${option}
+const handleCoffeeSizeInput = (coffeeSize) => {
+  showMessage(`A ${coffeeSize} ${option} coming up. Are you satisfied with your order?`, 'bot');
+  inputWrapper.innerHTML = //innerHTML input field.
+    `<input id="confirm-input" type="text" />
+  <button class="send-btn" id="send" type="submit">
+    Send
+  </button>`
+
+  //User gets input field to write answer after choosing size.
+  const inPutField = document.getElementById('confirm-input');
+
+
+  document.getElementById('send').addEventListener('click', () => handleConfirmInput(inPutField.value)) //Added eventhandler call-back (this one I had a reallt big issue to create) I remebered after 2 hours of trial and error that I needed .value
+  document.getElementById('confirm-input').addEventListener('keypress', (event) => {
+    if (event.key === 'Enter') handleConfirmInput(inPutField.value)
+  })
+
+}
+
+// This part of the code took me several hours to solve. My solution in the end was to take it row by row. What do I want to happend depending on answer in input field? 
+
+//Function answer in input. When user do or don't give valid answer in input field there will be different outcomes. 
+const handleConfirmInput = (confirmText) => {
+  showMessage(confirmText, 'user');
+
+  //First outcome - user answer with confirmText: yes
+  if (confirmText.toLowerCase() === 'yes') { //Added toLowerCase due to user and device compatibility. In Iphone for example, phone automatically begin words with capital letter.
```

Comment:
great, you really thought about everything! :)

Reply:
Thank you Jenny!